src/scraping/tools.py

```diff
@@ -253,3 +256,52 @@ def dataframe_to_db(self) -> None:
                     if_exists='replace',
                     index=False
                     )
+
+
+class GoogleSheets(DataBaseInteraction):
+    def __init__(self) -> None:
+        super().__init__()
+
+    @staticmethod
+    def base64_to_json(b64) -> dict[str, str]:
+        decodedbytes: bytes = base64.b64decode(b64[1:-1])
+        decodedstr: str = decodedbytes.decode("ascii")
+        json_dict: dict[str, str] = json.loads(decodedstr)
+        return json_dict
+
+    def read_from_sql(self, table_name: str) -> None:
+        self.db_connect()
+        df: pd.DataFrame = pd.read_sql(table_name, con=self.db_engine)
```

Comment:
maybe create another method that process the data?

Reply:
@oliverangelil had the great idea of checking the column type and making it automatic to change the date formats